### PR TITLE
Optional after

### PR DESCRIPTION
**Pull request by MrFizzyBubbs**
This would be nice to have for scripts that mostly operate in a predetermined order, such as community service or full day wrappers.

### CODE REVIEW

File: src/route.ts [hunk 1]
```diff
@@ -26,7 +26,21 @@ export function getTasks<T extends Task = Task>(quests: Quest<T>[]): T[] {
   return result;
 }
 
-export function orderByRoute<T extends Task = Task>(tasks: T[], routing: string[], ignore_missing_tasks?: boolean): Task[] {
+export function addExplicitAfter(tasks: Task[]): Task[] {
+  let lastImplicit;
+  for (const task of tasks) {
+    if (task.after !== undefined) continue;
```
Comment (Kasekopf) [inline src/route.ts]: Is it intended that this also skips the setting of lastImplicit? It seems most natural to me for this method to set the `after` of the current task to the previous task, regardless of if the previous task had an `after` or not.

Comment (MrFizzyBubbs) [reply src/route.ts]: It was when I wrote it but you make a good point.

**Comment by MrFizzyBubbs** [comment]
Alternatively, this could be implemented as an optional parameter of `getTasks`

**Comment by Kasekopf** [comment]
Yeah, actually I prefer the idea of adding a parameter to `getTasks` to include the explicit after settings.

**Pull request merged**